src/services/NotificationService.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore
+import {NotifyClient} from "notifications-node-client";
+import {Service} from "../models/injector/ServiceDecorator";
+import {NotifyClientMock} from "../../tests/models/NotifyClientMock";
+
+/**
+ * Service class for Certificate Notifications
+ */
+@Service()
+class NotificationService {
+    private readonly notifyClient: NotifyClient | NotifyClientMock;
+
+    constructor(notifyClient: NotifyClient | NotifyClientMock) {
+        this.notifyClient = notifyClient;
+    }
+
+    /**
+     * Sending email with the certificate according to the given params
+     * @param params - personalization details,email and certificate
+     */
+    public sendNotification(params: any) {
+        const emailDetails = {
+            personalisation: {
+                ...params.personalisation,
+                link_to_document: this.notifyClient.prepareUpload(params.certificate)
+            }
+        };
+
+        return this.notifyClient.sendEmail("1b602e0e-b53a-452a-858f-c5831ef3ed70", params.email, emailDetails)
```

Comment:
add the string to an enum file

Reply:
whats this hardcoded string required for? what is it?

Reply:
also can we pls add in some logs so that it becomes a bit easier to debug in prod as well. So log before sending email, with some details printed to identify what record we are emailing (just make sure we are not logging VIN/VRM and any Personal Identifiable Data

Reply:
it's the templateId we are sending to gov notify service. it's used to identify the specific template you want to use for your emails.

Reply:
done